packages/client/hmi-client/src/components/dataset/tera-save-dataset-from-simulation.vue

```diff
@@ -40,20 +32,13 @@ import { logger } from '@/utils/logger';
 
 const props = defineProps<{
 	simulationRunId?: string;
+	showDialog: boolean;
 }>();
 
 const saveAsName = ref('');
 const showSaveInput = ref<boolean>(false);
 const hasValidDatasetName = computed<boolean>(() => saveAsName.value !== '');
-const isSaveDisabled = computed<boolean>(() => {
-	if (
-		props.simulationRunId === undefined ||
-		props.simulationRunId === '' ||
-		!useProjects().activeProject.value?.id
-	)
-		return true;
-	return false;
-});
+const emit = defineEmits(['hide-dialog']);
 const saveDatasetToProject = async () => {
 	const { activeProject, refresh } = useProjects();
 	if (activeProject.value?.id) {
```

Comment:
you have removed checks on the `simulationRunId` which is possibly undefined.
we are now sending a
`logger.success(`Added dataset: ${saveAsName.value}`);` 
prematurely. we do not know if we successfully added it as if the simulation is undefined `saveDataset` will return false

Reply:
That check was move to where the menu option is defined

Reply:
can you point me to where it is being checked? I cannot see it

With the current approach i think `saveDatasetToProject` should logger.success if `saveDataset` returns true, and an error message when it returns false